src/Symfony/Component/Messenger/Stamp/LockStamp.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+use Symfony\Component\Lock\Key;
+
+final class LockStamp implements StampInterface
+{
+    private Key $key;
+    private ?float $ttl;
+    private bool $shouldBeReleasedBeforeHandlerCall;
+
+    public function __construct(
+        string $key,
+        ?float $ttl = 300.0,
```

Comment:
Let's use constructor property promotion here and for `$shouldBeReleasedBeforeHandlerCall`.

Reply:
Done